README.md

```diff
@@ -44,3 +44,121 @@ header_string, board_name, component_info, component_aliases = json2daisy.genera
 As you can see, the two functions above will return a number of supplementary objects in addition to the header string. These provide detailed information for how to interact with the generated header. The `component_info` dictionary provides getters and setters for I/O, and the `component_aliases` dictionary provides valid aliases for everything in `component_info`. 
 
 For examples of how to structure the board description, take a look at the JSON files in [src/json2daisy/resources](https://github.com/electro-smith/json2daisy/tree/main/src/json2daisy/resources).
+
+# pd2dsy JSON
```

Comment:
maybe don't mention pd2dsy here?

Reply:
nice catch! We're actively cleaning up the pd2dsy stuff.

Reply:
@stephenhensley I'm going to look at fully integrating json2daisy in hvcc/c2daisy later this year. this will also resolve an open debian packaging issue.

I will then probably move the documentation into the main hvcc documentation as well.

Assuming that json2daisy will no longer be used for any other projects and basically won't be needed as a separate library any more.

(I do want to look at finding a good way to retain the commit history somehow. I've split repositories in the past, but have never merged them like this so not sure if possible)

Reply:
@dromer that sounds great! I can get you added with necessary permissions to do a bit of maintenance here as you need to as well (since afaik this is soley used for hvcc/plug data at this point).

I haven't merged a repo, and its history, into another repo before before, but I know someone that just moved their entire multi-repo setup to a monorepo, and I think I remember them mentioning that they kept the history as well. 
I can pick their brain and see how crazy it was, or if I'm mistaken, and let you know.

Reply:
marking as resolved, the section now reads "Custom JSON"